Doc/howto/descriptor.rst

```diff
@@ -432,10 +869,9 @@ Using the non-data descriptor protocol, a pure Python version of
         def __init__(self, f):
             self.f = f
 
-        def __get__(self, obj, klass=None):
-            if klass is None:
-                klass = type(obj)
+        def __get__(self, obj, cls=None):
+            if cls is None:
+                cls = type(obj)
```

Comment:
The pure Python version doesn't capture the change https://github.com/python/cpython/pull/8405, which will call `self.f.__get__(cls)` if `self.f` has `__get__`.  I think this behavior is surprising (and a little weird) and should be captured in this document, which doesn't shy away from nitty gritty details.  I also happen to think this behavior of calling `f.__get__` [is incomplete](https://github.com/python/cpython/pull/22757) and should instead call `self.f.__get__(cls, cls)`.